### PR TITLE
A new type of Bottom Nav Bar added which is a bit similar to old one but looks awsome, however the old Nav Bar is still there.

### PR DESCRIPTION
**Pull request by Junaidhassan99**
To use the new Nav bar use a property gNavType in the GNav widget. Setting it to GNavType.defaultType renders the normal Nav bar while using GNavType.customType1 renders a new awsome Nav bar.

The new Nav Bar looks something like this:

Capture

Accepting this pull request will not affect the original functionality but it adds more features that a user can decide to have.

### CODE REVIEW

File: lib/src/button.dart [hunk 1]
```diff
@@ -51,6 +54,8 @@ class Button extends StatefulWidget {
   final Border? border;
   final Border? activeBorder;
   final List<BoxShadow>? shadow;
+  final GNavType gNavType;
```
[the remark this answers is not in the document]
Comment (Junaidhassan99) [reply lib/src/button.dart]: I have made all the requested changes to the repo you can accept the pull request now

File: lib/src/gnav.dart [hunk 2]
```diff
@@ -2,9 +2,9 @@ import 'package:flutter/material.dart';
 
 import 'gbutton.dart';
 
-enum GNavType {
+enum Style {
   defaultType,
-  customType1,
+  oldSchool,
 }
```
Comment (sooxt98) [inline lib/src/gnav.dart]: please change this to 
``` dart
enum GNavStyle {
    default,
    oldschool
}
```

Comment (Junaidhassan99) [reply lib/src/gnav.dart]: Requested change has been commited

File: lib/src/gnav.dart [hunk 3]
```diff
@@ -3,7 +3,7 @@ import 'package:flutter/material.dart';
 import 'gbutton.dart';
 
 enum Style {
```
Comment (sooxt98) [inline lib/src/gnav.dart:5]: this need to change as well, GnavStyle

Comment (Junaidhassan99) [reply lib/src/gnav.dart]: requested change have been commited

**Comment by sooxt98** [comment]
Could you please fix the example, seems its outdated 
https://github.com/sooxt98/google_nav_bar/blob/60b8927667293aa71e1fae66e07c66f6d16f7b27/example/lib/main.dart#L76

**Pull request merged**